code/modules/mining/lavaland/loot/ashdragon_loot.dm

```diff
@@ -191,6 +191,14 @@
 	if(is_type_in_typecache(target, banned_turfs))
 		return
 
+	if(!is_mining_level(user.z)) //Will only spawn a few sparks if not on mining z level
+
+
+		timer = world.time + create_delay + 1
+		user.visible_message("<span class='danger'>[user]'s lava staff malfunctions!</span>")
```

Comment:
```suggestion
		user.visible_message("<span class='danger'>[user]'s [src] malfunctions!</span>")
```
Will need to check if this formats correctly. If not try using `name` instead of `src`